### ISSUE TITLE
Plot suggestion

### ISSUE THREAD

**Issue by David-Levinson** [COVID19Tracking/covid-tracking-dash#2]
I'd really think it is useful if we could see a % of positive tests each day based on the total number of tests being conducted.  I did this for one state and the data was erratic but I think it will be helpful in the long run and will help normalize the data as the amount of testing increases.
Currently it's hard to tell how the infections are spreading or if it's just an increase in testing.  It's both of course but we need to a way to normalize it.

**Comment by eric-czech** [comment]
Thanks @chrisroberts7577, there was some discussion about that in the Slack group for the project and the consensus was to avoid it.  The most convincing reason I caught was that the fraction can skew heavily based on which populations states are targeting with testing.  Hospitalizations, possibly per capita, were deemed a better indication of how quickly conditions are deteriorating so I added that instead now that the project is collecting it.

**Comment by eric-czech** [comment]
Oh and btw @chrisroberts7577, @sharmanirek had added that metric in https://github.com/COVID19Tracking/covid-tracking-dash/pull/1 if you wanted to look at the metric anyhow.  I did and thought that the unreliability of it wasn't worth dealing with the display issues relating to showing a percentage/rate along with all the other unbounded counts (the current scales and rounding aren't compatible). I nixed it but it's there if anybody finds this and wants to see it regardless.
